dev_tools.coffee

```diff
@@ -34,8 +34,8 @@ else if program.change
 
     if program.fields
         konnectorName = program.change
-        fields = program.fields.split(',')
-        log.info "Changing fields for #{konnectorName} with #{fields}."
+        fields = JSON.parse(program.fields)
```

Comment:
Major change in dev tools, I require the `fields` attribute to be a JSON string describing the fields to set. This was the easiest way to get the dev tools to let me set the weboob konnectors required fields. But that may not be something we want to merge as is.